src/participant/participant.ts

```diff
@@ -32,6 +35,7 @@ const NUM_DOCUMENTS_TO_SAMPLE = 3;
 interface ChatResult extends vscode.ChatResult {
   metadata: {
     responseContent?: string;
+    responseType: ParticipantResponseType;
```

Comment:
When leaving feedback for a response, we don't have access to the original prompt, which is why we need to supply the response type as we're creating the result. This is only currently used for telemetry reporting.